client/src/components/Overview/Overview.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
+import PrimaryImage from './PrimaryImage.jsx';
+import sampleData from './sampleData';
+
+var Overview = ({ productId }) => {
+
+  const [product, getProduct] = useState(sampleData);
```

Comment:
Do we still need the reference to sampleData here?

Reply:
Sample data has been removed.